### DISCUSSION TITLE
Self-Sovereign Identity Integration

### DISCUSSION THREAD

**Discussion by fforootd** [zitadel/zitadel#1352] [Ideas]
We should discuss how we want to integrate SSI into ZITADEL in the future

- [ ] Should we also be able to issue credentials?
- [ ] Check different protocols and Architectures
- [ ] Integrate native or with protocols like OIDC SIOP?

**Comment by gc-ss** [comment]
>  Integrate native or with protocols like OIDC SIOP?

There has been some interest starting at https://gitter.im/hashicorp-nomad/Lobby?at=610044086ccf813819debfc6

**Reply by fforootd** [reply]
Well, we plan to support multiple protocols as soon as it shows which will do the "game"
